google/cloud/aiplatform/training_jobs.py

```diff
@@ -288,11 +294,8 @@ def _run_job(
                 than the one given on input. The output URI will
                 point to a location where the user only has a
                 read access.
-            training_task_inputs (dict):
-                Required. The training task's parameter(s), as specified in
-                the
-                ``training_task_definition``'s
-                ``inputs``.
+            training_task_inputs (proto.Message):
```

Comment:
Since we pass in the proto, it'd be nice if we could just remove the `training_task_definition yaml file` as the proto basically represents the same thing. 

Since the service still needs this passed in, we could use a dictionary to infer the correct `training_task_definition yaml file URI`.

@sasha-gitg thoughts?

Reply:
Okay good to know. It will be cleaner to keep the concepts separate then.